change/@ni-nimble-components-56124785-d76b-4bd0-b330-b457eb4ff9a0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@m-akinc fyi some of the armstrong folks were also interested in the status of the PR. Still in draft / has open comments / reviewers not reset so assuming still in work and not waiting on reviews.

Reply:
Just now got the names finalized and let Malcolm know it's ready for buddy.